meson.build

```diff
@@ -4,6 +4,10 @@ run_command('bin/ksh93_prereq.sh')
 
 add_project_arguments('-D_PACKAGE_ast', language: 'c')
 add_project_arguments('-DMESON_BUILD=1', language: 'c')
+# This was added because OpenBSD installs a lot of stuff in /usr/local/{include,lib}
+# that would normally be in /usr/{include,lib}. But the compiler doesn't
+# automatically search /usr/local/included for headers so force it to do so.
+add_project_arguments('-I/usr/local/include', language: 'c')
```

Comment:
If it is only needed on OpenBSD then we should check for [host machine](http://mesonbuild.com/Reference-manual.html#build_machine-object) and put this code block under `if` condition.

Reply:
I'll do that but that still means adding the arg on other BSDs like FreeBSD. But that's probably still safer than doing it unconditionally for every platform. On the other hand there does seem to be a trend among newer distros to break from the past with respect to things like /usr/lib.